### PR TITLE
#vp-635 Getting started

### PR DESCRIPTION
**Pull request by mvktsk**
Getting started md created

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Getting started
+
+If you are new to Virto Commerce, follow these steps to set up and launch your store:
+
+## Configure your store
```
Comment (eMazeika) [inline docs/getting-started.md]: move down, after catalog

File: docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Order management
+
+Virto Commerce provides the following tools for order management:
+
+* [Working with Payments]()
+* [Working with Shipments](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/order-management/working-with-shipments.md)
+* [Working with Taxes](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/order-management/working-with-taxes.md)
+
+## Marketing & promotional tools
```
Comment (eMazeika) [inline docs/getting-started.md]: move up, before orders

File: docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+Additionally you can read [Working with Products catalog](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md) article. It describes how to work with Products [catalog](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#common-catalogs), [categories](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#categories), [items](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#items-products), product [properties](Properties), product [variations](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#variations) and [virtual catalogs](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#virtual-catalogs)
+
+## Order management
+
+Virto Commerce provides the following tools for order management:
+
+* [Working with Payments]()
```
[the remark this answers is not in the document]
Comment (mvktsk) [reply docs/getting-started.md]: As designed :)

File: docs/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+Virto Commerce platform is 4-MULTI: multi-language, multi-currency, multi-theme and multi-store by design. This gives the ability to have and run multiple stores on the same system.
+
+To configure your store you must accomplish a set of actions:
+
+1. Create first store
+2. Select your catalog
+3. Select available currency, languages
+4. Upload theme
+5. …
```
Comment (OlegoO) [inline docs/getting-started.md]: @mvktsk  Need to select right steps and remove ... I just added as example

File: docs/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+6. Rebuild catalog index
+
+Check that new store is visible.
+
+The [Managing stores](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/configuration/store.md) article helps you to configure your first store. Also at this step you should configure [Users, Roles and Role Assignment](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/users-management-roles-and-role-assignment.md)
+
+## Step 3. Configure prices and inventory
```
Comment (OlegoO) [inline docs/getting-started.md]: Define simple actions: 
like create price list, add products, create assignment

File: docs/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+Price list assignment identifies the price list from which a product price is shown for customers. Moreover, price list assignments can do much more than simply provide default list and sale prices for products. This feature allows you to provide custom prices to targeted customers according to customer's characteristics such as age, gender, geographic location and searched terms.
+
+After this step completed, you can [Open new products in Virto Commerce Storefront](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/lessons/lesson2.md#open-new-products-in-virto-commerce-storefront-frontend) , browse the catalog, add product to shopping cart and create order. You can view customer order via admin UI.
+
+## Step 4. Configure payments, shipments, taxes
+
+By default, you see demo payment and shipmen methods. Virto Commerce provides the ready to use module which. You can set your own configurations:
+
+* Configure [Payments](TODO) method
```
Comment (OlegoO) [inline docs/getting-started.md]: Missed
1. Install Payment module (For example: ABC with link)
Remove broken links

...

File: docs/getting-started.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+## Step 5. Launch Marketing & promotional tools
+
+Marketing your store is essential to gain your customers loyalty and raise awareness among potential customers.
+
+There are several marketing tools in Virto Commerce: you can publish content, create personalized shopping experiences for customers, manage promotions.
+
+Start with Creation of two simple promotions:
+
+1. 5% if cart subtotal > 100 $
```
Comment (OlegoO) [inline docs/getting-started.md]: Add more details: Added  more details

File: docs/getting-started.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+* [Create a content publishing](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/marketing/how-to-add-an-advertising-spot-via-marketing/create-a-content-publishing.md)
+* [Dynamic content](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/marketing/dynamic-content.md)
+* [Managing promotions](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/marketing/promotions.md)
+* [Combine active promotions](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/marketing/combine-active-promotions.md)
+
+## Step 6. Configure User Experience
+
+To enhance User Experience Virto Commerce contains a set of tools:
+
+* [Creating a new menu item in Virto Commerce Storefront](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/lessons/lesson2.md#creating-a-new-menu-item-in-virto-commerce-storefront-frontend)
```
Comment (OlegoO) [inline docs/getting-started.md]: Review an add other activities which we already described in docs

File: docs/getting-started.md (hunk 4 above)
Comment (mvktsk) [reply docs/getting-started.md]: Fixed

File: docs/getting-started.md (hunk 5 above)
Comment (mvktsk) [reply docs/getting-started.md]: fixed

File: docs/getting-started.md (hunk 6 above)
Comment (mvktsk) [reply docs/getting-started.md]: fixed

File: docs/getting-started.md (hunk 7 above)
Comment (mvktsk) [reply docs/getting-started.md]: Didn't get what needs to be done, need more details :)

File: docs/getting-started.md (hunk 8 above)
Comment (mvktsk) [reply docs/getting-started.md]: need your help to find activities

File: docs/getting-started.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
+---
+date: '2019-10-22'
```
[the remark this answers is not in the document]
Comment (mvktsk) [reply docs/getting-started.md]: I suggest taking screenshots from version 3.0 release. And create issue in the backlog for the task

[the remark this answers is not in the document]
Comment (asvishnyakov) [reply docs/getting-started.md]: I think we need something like tabs for images thumbnails and duplicate all of them for v2 and v3

File: docs/getting-started.md [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+1. Create first store
+2. Select your catalog
+3. Select available currency, languages
+4. Configure advanced properties:
+    1. Store description, emails and URLs
+    2. Fulfillment centers
+    3. Operational timezone
+5. Setup theme
+
+Check that new store is visible.
```
Comment (eMazeika) [inline docs/getting-started.md]: what's visible where? in the list?

File: docs/getting-started.md [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+
+The [Managing stores](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/configuration/store.md) article helps you to configure your first store. Also at this step you should configure [Users, Roles and Role Assignment](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/users-management-roles-and-role-assignment.md)
+
+## Step 3. Configure prices and inventory
+
+In the Virto Commerce, a price list is a set of pricing details that can be assigned to items. All prices are stored in a price list, which is associated with a catalog and a currency. When the price list is created in the Virto Commerce, you need to define which products will be included into this price list and what prices will be defined for them. You can do it by creating price list assignment.
```
Comment (eMazeika) [inline docs/getting-started.md]: In Virto Commerce, a Pricelist is a set of pricing details that can be assigned to items. All prices are stored in the pricelist(s), which have specific currency set. When a pricelist is created, you define which products will be included in this list and what prices will be defined for them.
The way to apply the prices to products on Storefront is creating a Pricelist Assignment. It binds a catalog with pricelist and can add optional conditions for this binding.

`
Also update the initial source of this text, if there was any
`

File: docs/getting-started.md [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+2. Add items to the pricelist
+
+3. Create price list assignment
+4. Manage inventory
+
+After this step completed, you can [Open new products in Virto Commerce Storefront](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/lessons/lesson2.md#open-new-products-in-virto-commerce-storefront-frontend) , browse the catalog, add product to shopping cart and create order. You can view customer order via admin UI.
+
+## Step 4. Configure payments, shipments, taxes
+
+By default, you see demo payment and shipmen methods. Virto Commerce provides the ready to use module which. You can set your own configurations:
```
Comment (eMazeika) [inline docs/getting-started.md]: shipmen**t**
modules, which you...

File: docs/getting-started.md (hunk 7 above)
Comment (eMazeika) [reply docs/getting-started.md]: Could it be: "Each customer would receive a 5 percent off if cart subtotal exceeded $100."?
Add more details to the second promotion as well.

File: docs/getting-started.md [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+
+* [Menu item in Virto Commerce Storefront](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/lessons/lesson2.md#creating-a-new-menu-item-in-virto-commerce-storefront-frontend)
+* Catalog filters in Virto Commerce Storefront
+* Currencies, languages
+* Link lists
+* Theme settings file
+* Login on behalf
+* Social login
+
+## Step 7. Go to live with MVP and review additional cases and Virto Commerce
```
Comment (eMazeika) [inline docs/getting-started.md]: 'in Virto Commerce'???

File: docs/getting-started.md (hunk 1 above)
Comment (mvktsk) [reply docs/getting-started.md]: fixed

File: docs/getting-started.md (hunk 2 above)
Comment (mvktsk) [reply docs/getting-started.md]: As designed :)

**Comment by mvktsk** [comment]
@eMazeika 
> price list => pricelist everywhere

In docs we use `price list` https://virtocommerce.com/docs/vc2userguide/price-lists

File: docs/getting-started.md (hunk 7 above)
Comment (mvktsk) [reply docs/getting-started.md]: fixed

File: docs/getting-started.md (hunk 8 above)
Comment (mvktsk) [reply docs/getting-started.md]: fixed

File: docs/getting-started.md [hunk 14]
```diff
@@ -0,0 +1,102 @@
...
+To configure your store you must accomplish a set of actions:
+
+1. Create first store
+2. Select your catalog
+3. Select available currency, languages
+4. Configure advanced properties:
+    1. Store description, emails and URLs
+    2. Fulfillment centers
+    3. Operational timezone
+5. Setup theme
```
[the remark this answers is not in the document]
Comment (mvktsk) [reply docs/getting-started.md]: fixed in this way 
1. Create first store (enter store code and name)
2. Select your catalog
3. Select available currency, languages
4. Configure advanced properties:
    1. Store description, emails and URLs
    2. Fulfillment centers
    3. Operational timezone
5. Setup theme

unfortunately, I did not find the link to the good Setup theme topic in our docs

File: docs/getting-started.md [hunk 15]
```diff
@@ -0,0 +1,102 @@
...
+* Create a product with variants
+
+Additionally you can read [Working with Products catalog](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md) article. It describes how to work with products [catalog](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#common-catalogs), [categories](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#categories), [items](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#items-products), product [properties](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#properties)), product [variations](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#variations) and [virtual catalogs](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#virtual-catalogs)
```
[the remark this answers is not in the document]
Comment (mvktsk) [reply docs/getting-started.md]: Fixed

File: docs/getting-started.md (hunk 12 above)
Comment (mvktsk) [reply docs/getting-started.md]: fixed

File: docs/getting-started.md (hunk 13 above)
Comment (mvktsk) [reply docs/getting-started.md]: fixed

File: docs/getting-started.md (hunk 10 above)
Comment (mvktsk) [reply docs/getting-started.md]: fixed in this way
`Open your first store in a browser.`

File: docs/getting-started.md (hunk 11 above)
Comment (mvktsk) [reply docs/getting-started.md]: fixed 
In docs we use `price list` https://virtocommerce.com/docs/vc2userguide/price-lists

Comment (mvktsk) [reply docs/getting-started.md]: price list => pricelist fixed everywhere

File: docs/getting-started.md [hunk 16]
```diff
@@ -0,0 +1,104 @@
...
+3. Create pricelist assignment
+4. Manage inventory
+
+After this step completed, you can [Open new products in Virto Commerce Storefront](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/lessons/lesson2.md#open-new-products-in-virto-commerce-storefront-frontend) , browse the catalog, add product to shopping cart and create order. You can view customer order via admin UI.
+
+## Step 4. Configure payments, shipments, taxes
+
+By default, you see demo payment and shipment methods. Virto Commerce provides the ready to use module which you can set your own configurations:
+
+* Install Payment module (For example: PayPal)
```
Comment (eMazeika) [inline docs/getting-started.md]: link to paypal: https://virtocommerce.com/apps/extensions/paypal
Also, to demo payment: https://github.com/VirtoCommerce/vc-module-payment

File: docs/getting-started.md [hunk 17]
```diff
@@ -0,0 +1,104 @@
...
+* Configure [Shipments](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/order-management/working-with-shipments.md) method
```
Comment (eMazeika) [inline docs/getting-started.md]: Just not sure, if ShipStation integration module is in scope of "Getting started". @OlegoO  ?
Maybe, link to https://github.com/VirtoCommerce/vc-module-shipping is enough?

File: docs/getting-started.md [hunk 18]
```diff
@@ -0,0 +1,104 @@
...
+* Configure [Taxes](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/order-management/working-with-taxes.md) method
```
Comment (eMazeika) [inline docs/getting-started.md]: Again, https://github.com/VirtoCommerce/vc-module-tax should be enough, I think

File: docs/getting-started.md [hunk 19]
```diff
@@ -0,0 +1,104 @@
...
+## Step 5. Launch Marketing & promotional tools
+
+Marketing your store is essential to gain your customers loyalty and raise awareness among potential customers.
+
+There are several marketing tools in Virto Commerce: you can publish content, create personalized shopping experiences for customers, manage promotions.
+
+Start with creation of two simple promotions:
+
+1. Each customer would receive a 5%  off if cart subtotal exceeded $100
+2. Add Gift product to shopping cart for every fifth customer order
```
Comment (eMazeika) [inline docs/getting-started.md]: interesting, but i don't think we currently support scenario like this

Comment (mvktsk) [reply docs/getting-started.md]: Could it be `Add Gift product to shopping cart for a limited period of time` ?

File: docs/getting-started.md (hunk 16 above)
Comment (mvktsk) [reply docs/getting-started.md]: fixed

File: docs/getting-started.md (hunk 18 above)
Comment (mvktsk) [reply docs/getting-started.md]: Fixed

File: docs/getting-started.md (hunk 19 above)
Comment (mvktsk) [reply docs/getting-started.md]: Fixed

File: docs/getting-started.md (hunk 17 above)
Comment (mvktsk) [reply docs/getting-started.md]: unfortunately, there are no descriptions in https://github.com/VirtoCommerce/vc-module-shipping

**Pull request merged**